spec/amp-story-player.md

```diff
@@ -126,3 +130,179 @@ CSS variable with the URL pointing to the poster image of the story.
   </a>
 </amp-story-player>
 ```
+
+#### exit-control
+
+Set to `back-button` or `close-button`. The button will dispatch an `amp-story-player-back` or `amp-story-player-close` when clicked. This button will disappear when a page attachment is open and reappear when closed.
+
+## Programmatic Control
+
+Call the player's various methods to programmatically control the player.
```

Comment:
I'd like to add that the methods are exposed on the DOM HTML element and also in the global class variable. Not sure where would be the right place, maybe here?